### ISSUE TITLE
RTL export broken

### ISSUE THREAD

**Issue by Man-with-Arrow** [giuspen/cherrytree#2044]
**Version, Operative system**
v0.99.47, Fedora 35. Bug has been present since I started using Cherrytree on Debian 8, around 2015.

**Describe the bug**
Documents in RTL script are exported as LTR.

**To Reproduce**
Create a new node with any RTL script. Note that Cherrytree renders it perfectly fine.
Export to PDF or HTML. Notice that it's now LTR.

**Screenshots**
![Screenshot from 2022-05-14 15-46-56](https://user-images.githubusercontent.com/36547425/168426379-c1a6ae9a-6bf2-403c-a172-e2f0b48d520c.png)
![Screenshot from 2022-05-14 15-47-34](https://user-images.githubusercontent.com/36547425/168426380-92eae8d7-bb6a-4495-903e-d1e750081cf6.png)

**Potential Solution**

After exporting to HTML, adding `dir=rtl` to the exported HTML's <head> makes the whole file RTL, and breaks LTR paragraphs. A solution could be to add the `dir` attribute to each paragraph on export, according to the detected alignment in Cherrytree itself.

**Comment by giuspen** [comment]
can you attach to the issue a sample cherrytree document with mixed RTL and LTR text so I'll try and fix at least the html export

**Comment by Man-with-Arrow** [comment]
[Of course.](https://github.com/giuspen/cherrytree/files/8747373/sample.zip) Please let me know if there's anything else I can do to help!

**Comment by giuspen** [comment]
Yes I'm getting there with the pdf too, even though much more difficult than the html. Good that you keep an eye, so when I'll declare it complete some testing will help. I think it will take few more commits.
